build.sbt

```diff
@@ -35,7 +35,14 @@ val sharedSettings = Project.defaultSettings ++ scalariformSettings ++  Seq(
 
   javacOptions ++= Seq("-target", "1.6", "-source", "1.6"),
 
-  scalacOptions ++= Seq("-unchecked", "-deprecation", "-optimize", "-Xlint", "-language:implicitConversions", "-language:higherKinds", "-language:existentials"),
+  scalacOptions ++= Seq(
+    "-unchecked",
+    "-deprecation",
+    "-optimize",
+    "-Xlint:-adapted-args",
```

Comment:
`scala -Xlint:help` says that `adapter-args` is for "Warn if an argument list is modified to match the receiver." Is that to warn on implicit conversions?